api/src/controllers/error.ts

```diff
@@ -47,7 +47,7 @@ export const errorHandler: ErrorRequestHandler = (err, req, res, next) => {
     res.status(err.status).json(err.responseJson);
   } else {
     // Log this to prevent it getting swallowed
-    log.error(err);
+    console.error(err);
```

Comment:
Can get rid of this now after syncing with main, as was fixed by https://github.com/wellcomecollection/content-api/pull/74/files